mypyc/genops.py

```diff
@@ -1839,29 +1848,31 @@ def add_call_to_callable_class(self, call_fn: FuncIR) -> None:
         callable_class = self.fn_info.callable_class
         callable_class.methods['__call__'] = call_fn
 
-    def instantiate_callable_class(self, fdef: FuncDef) -> Value:
+    def instantiate_callable_class(self) -> Value:
         """Assigns a callable class to a register named after the given function definition."""
         fullname = '{}.{}'.format(self.module_name, self.fn_info.callable_class.name)
-        func_reg = self.add(Call(RInstance(self.fn_info.callable_class), fullname, [], fdef.line))
+        func_reg = self.add(Call(RInstance(self.fn_info.callable_class), fullname, [],
+                            self.fn_info.fdef.line))
```

Comment:
And I *definitely* don't think that replacing all uses of `fdef` with the much more verbose `self.fn_info.fdef` makes the code easier to follow. At least pull that one out into a variable in some of these noisy places.